src/HotbizApi/handler.py

```diff
@@ -0,0 +1,97 @@
+from dotenv import load_dotenv
+
+import json
+import os
+
+import boto3
+from boto3.dynamodb.conditions import Key, Attr
+
+# Hotspot will return a list of the
+# most heavily trafficked regions in
+# the configured region
+def hotspot(event, context):
+    # Prepares our database client
+    # to read prepared results from 
+    # the `hotspots` table
+    db = DynamoAccessor("hotspots")
+
+    # User-provided options
+    geo = event['queryparameters']['geofence']
+    date = event['queryparameters']['date']
+
+    # gather the results
+    data = db.get_all("hotspots", 10, date)
+
+    print(data)
+
+    return {
+            "statusCode": 200,
+            "body": json.dumps(
+                data
+            ),
+            "headers": {
+                "Content-Type": "application/json"
+            }
+    }
+
+class DynamoAccessor():
```

Comment:
Wound up being less important for this PoC, but for an actively maintained project, would be interested in seeing how dependency injection could be implemented (in a dynamically-typed language) across production / unit testing environments